File: code_blocks/integrations/third-party-integrations/iterable_1.kt
import com.iterable.iterableapi.IterableApi
import com.revenuecat.purchases.Purchases

Purchases.configure(this, "public_sdk_key", appUserID = "my_app_user_id")
IterableApi.initialize(this, "<YOUR_API_KEY>")

IterableApi.setEmail("user@example.com")
IterableApi.setUserId("user123")

Purchases.sharedInstance.setAttributes(mapOf(
    "\$email" to "user@example.com",
    "\$iterableUserId" to "user123",
    "\$iterableCampaignId" to "123",
    "\$iterableTemplateId" to "123"
))

File: code_blocks/integrations/third-party-integrations/iterable_1.dart
import 'package:purchases_flutter/purchases_flutter.dart';
import 'package:iterable_flutter/iterable_flutter.dart';

Future<void> setup() async {
  await Purchases.configure(
    PurchasesConfiguration('public_sdk_key')..appUserId = 'my_app_user_id',
  );
  await IterableFlutter.initialize('<YOUR_API_KEY>');
  await IterableFlutter.setEmail('user@example.com');
  await IterableFlutter.setUserId('user123');
  await Purchases.setAttributes({
    '\$email': 'user@example.com',
    '\$iterableUserId': 'user123',
    '\$iterableCampaignId': '123',
    '\$iterableTemplateId': '123',
  });
}

